projectDocs/Design_Implementation.md

```diff
@@ -0,0 +1,54 @@
+# Design (Subject to updates)
+## Model  
+### Database  
+|   Database column   |   expected input   |        Thoughts on implementation        |
+| ------------------- | ------------------ | ---------------------------------------- |
+|          pk         |    auto-generate   |                                          |
+|       subject       |  string from user  | short description to user to add clarity |
+|  Issue comment FK   |  string from user  | long description to user to add clarity  |
+|     contactName     |  string from user  |                                          |
+|    dateSubmitted    |    auto-generate   |                                          |
+|     issueStatus     | Pulled from github |                                          |
+| github issue number | Pulled from github |                                          |
+|  github repository  | pulled from github |           probably don't need            |
+       
+ ### Reference Codes  
+  |  statusCode  |      StatusLabel      |  
+  | ------------ |  -------------------- |  
+  |      10      |  "Request Received"   |  
+  |      20      |  "Problem Confirmed"  |  
+  |      30      |  "Fix Defined"        |  
+  |      40      |  "Fix Implemented"    |  
+  |      50      |  "Fix Confirmed"      |  
+    
+    
+- Separate comment database will have:  (hard to tell if we need this)
+      - pk
+      - issueFK
+      - date commented
+      - Comment
+      
+### Comment database
+| Database Column |     Expected input     | Thoughts on implementation |
+| --------------- | ---------------------- | -------------------------- |
+|        pk       |     auto-generated     |                            |
+|     issueFK     |     auto-generated     |                            |
+| date commented  |     auto-genereted     |                            |
+|     comment     | passed from controller |                            |
+    
+## Controller
+- Student/Staff submit new issue
+  - login is optional, login with D&E info else anonymous
+    - If anonymous then ask for contact details?
+  - Restful API to database 
+- Should be able to view/add comments to it.  CAN NOT update original issue.
```

Comment:
There are more details to this. DevOps staff don't need to be able to read/edit data in the app; they will do that at GitHub and it will sync. Other users should be able to view and add comments for Issues that *they created*, but not access other users' Issues.